numpy/distutils/fcompiler/gnu.py

```diff
@@ -193,13 +199,10 @@ def get_libraries(self):
             opt.append(g2c)
         c_compiler = self.c_compiler
         if sys.platform == 'win32' and c_compiler and \
-               c_compiler.compiler_type == 'msvc':
+                c_compiler.compiler_type == 'msvc':
             # the following code is not needed (read: breaks) when using MinGW
             # in case want to link F77 compiled code with MSVC
             opt.append('gcc')
-            runtime_lib = msvc_runtime_library()
```

Comment:
Question: this change is in the class for ``g77``, so is this a cleanup that looks right or have you been using ``g77``?

Comments:
- the code comment above about MinGW looks like it applied to these lines and should be removed.
- the import of ``msvc_runtime_library`` is now unused and should be removed.